scipy/stats/_distribution_infrastructure.py

```diff
@@ -393,15 +402,23 @@ class _RealDomain(_SimpleDomain):
         and having value NaN are specified by `proportions`.
 
     """
+    def _get_endpoint_str(self, endpoint, funcname):
+        if callable(endpoint):
+            if endpoint.__doc__ is not None:
+                return endpoint.__doc__
+            params = inspect.signature(endpoint).parameters.values()
+            params = [
+                p.name for p in params if p.kind == inspect.Parameter.KEYWORD_ONLY
+            ]
+            return f"{funcname}({','.join(params)})"
+        return self.symbols.get(endpoint, f"{endpoint}")
 
     def __str__(self):
```

Comment:
There are some tests of the `__str__` method of `_RealDomain` that maybe could be updated:
https://github.com/scipy/scipy/blob/0b91033ba51a8756de2c4b3ce4c1e6dfeb3884eb/scipy/stats/tests/test_continuous.py#L108

Could also add oddball things to
https://github.com/scipy/scipy/blob/0b91033ba51a8756de2c4b3ce4c1e6dfeb3884eb/scipy/stats/tests/test_continuous.py#L1713

if they don't fit elsewhere.